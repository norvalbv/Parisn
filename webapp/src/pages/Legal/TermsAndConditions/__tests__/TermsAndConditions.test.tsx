import { FC } from 'react';
import renderer from 'react-test-renderer';

import TermsAndConditions from '../';

describe('<TermsAndConditions />', () => {
  test('TermsAndConditions props with mandatory props', () => {
    const tree = renderer.create(<TermsAndConditions />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
