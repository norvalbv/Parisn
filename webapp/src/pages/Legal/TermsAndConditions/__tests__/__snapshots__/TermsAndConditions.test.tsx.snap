// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TermsAndConditions /> TermsAndConditions props with mandatory props 1`] = `
<div
  className="h-screen w-full grid place-items-center bg-primary-dark text-primary-light"
>
  <div
    className=" w-1/2"
  >
    <p
      className="text-center"
    >
       
      Lorem ipsum dolor sit, amet consectetur adipisicing elit. Quas et numquam similique molestiae suscipit laudantium ducimus doloribus itaque soluta? Odio aspernatur, rem illo dolores quasi debitis perspiciatis eos deserunt. Sequi quia fugit, maxime id debitis at corporis facilis non laborum quod illum. Impedit quisquam quis sapiente ratione voluptatem, natus cupiditate culpa voluptates animi asperiores provident nemo ducimus nesciunt repudiandae optio odit rerum pariatur molestiae vel. Explicabo culpa molestiae at, possimus quaerat officiis, quidem dolores deleniti a distinctio facere incidunt, ipsa cupiditate libero fugit dolorem sint! Dolores numquam consequatur veritatis cupiditate, dolore commodi, blanditiis ut ab dolor nobis praesentium, nostrum maiores aliquam iusto. Accusamus, repellat. Eveniet laborum maxime adipisci inventore impedit voluptatibus nobis repellat ipsa quibusdam temporibus sapiente numquam ratione cupiditate voluptate quo, blanditiis quos hic. Voluptatem itaque praesentium dolores esse numquam omnis perferendis saepe. Praesentium debitis quae inventore sequi quo modi in minus perspiciatis magni, laudantium reiciendis nam fugiat et labore? Pariatur nisi recusandae sint velit fugit, necessitatibus sunt temporibus incidunt blanditiis natus deserunt facilis excepturi laudantium corrupti. Vitae fugit odit, voluptas eaque aliquam accusantium fuga voluptatibus explicabo ipsa maxime praesentium animi minus incidunt in, alias voluptate ipsam reprehenderit nostrum tenetur. Quidem repudiandae animi quos saepe culpa. Quas, repellat tempora.
    </p>
  </div>
</div>
`;
